src/main/kotlin/org/koitharu/kotatsu/parsers/site/vi/BlogTruyenParser.kt

```diff
@@ -20,7 +20,7 @@ class BlogTruyenParser(context: MangaLoaderContext) :
 	PagedMangaParser(context, MangaParserSource.BLOGTRUYEN, pageSize = 20) {
 
 	override val configKeyDomain: ConfigKey.Domain
-		get() = ConfigKey.Domain("blogtruyenmoi.com")
+		get() = ConfigKey.Domain("blogtruyenmoi.com", "blogtruyen.vn")
```

Comment:
The 2 urls link to different pages, do you have any information to make sure you know the same source?

Reply:
i can sure that those 2 domains are the same source (or the same backend), just 2 different hosts. I tried changing both domains and it still works very well

Reply:
Of course it's likely to work, because it uses the same template buy online, so even if another url works it doesn't mean they're the same.
If you're not sure, it's best to add it as a new source.

Reply:
> Of course it's likely to work, because it uses the same template buy online, so even if another url works it doesn't mean they're the same. If you're not sure, it's best to add it as a new source.

i can confirm that because im a follower of blogtruyen in Vietnam